parser/Builder.cc

```diff
@@ -1784,7 +1784,7 @@ class Builder::Impl {
         // Only report an error if this name doesn't start with an `_`, but still report it as being a duplicate of
         // another argument, for the purpose of mangling. The Ruby VM silences these errors.
         if (this_name.shortName(gs_)[0] != '_') {
-            error_without_recovery(ruby_parser::dclass::DuplicateArgument, this_loc, this_name.toString(gs_));
+            error(ruby_parser::dclass::DuplicateArgument, this_loc, this_name.toString(gs_));
```

Comment:
Do we have a plan for what to do about other uses of `error_without_recovery`? Because probably the same problem that this PR addresses also applies to other usages.

Reply:
I was planning to go through and audit the other uses of `error_without_recovery` after landing this change, to sort out whether or not the remaining uses really needed to trigger that behavior.